AssertionError: getItemComposition must be called on client thread

When you try to inspect someone with assertions enabled, you receive the following error:

```
[AWT-EventQueue-0] ERROR net.runelite.client.RuneLite - Uncaught exception:
java.lang.AssertionError: getItemComposition must be called on client thread
	at net.runelite.client.game.ItemManager.getItemComposition(ItemManager.java:386)
	at net.runelite.client.game.ItemManager.getItemPrice(ItemManager.java:308)
	at net.runelite.client.game.ItemManager.getItemPrice(ItemManager.java:287)
	at equipmentinspector.EquipmentInspectorPanel.lambda$null$0(EquipmentInspectorPanel.java:99)
	at java.util.HashMap.forEach(HashMap.java:1289)
	at equipmentinspector.EquipmentInspectorPanel.lambda$update$1(EquipmentInspectorPanel.java:96)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:758)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:74)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:728)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:205)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

--- a/src/main/java/equipmentinspector/EquipmentInspectorPanel.java
+++ b/src/main/java/equipmentinspector/EquipmentInspectorPanel.java
@@ -74,10 +74,10 @@ public class EquipmentInspectorPanel extends PluginPanel
                 .addComponent(equipmentPanels)
         );
 
-        update(new HashMap<>(), "");
+        update(new HashMap<>(), new HashMap<>(), "");
     }
 
-    public void update(Map<KitType, ItemComposition> playerEquipment, String playerName)
+    public void update(Map<KitType, ItemComposition> playerEquipment, Map<KitType, Integer> equipmentPrices, String playerName)
     {
 
         if (playerName.isEmpty() || playerName == null)
@@ -96,8 +96,8 @@ public class EquipmentInspectorPanel extends PluginPanel
                     playerEquipment.forEach((kitType, itemComposition) ->
                     {
                         AsyncBufferedImage itemImage = itemManager.getImage(itemComposition.getId());
-                        int GetitemPrice = itemManager.getItemPrice(itemComposition.getId());
-                        totalItemPrice.addAndGet(itemManager.getItemPrice(itemComposition.getId()));
+                        int GetitemPrice = equipmentPrices.get(kitType);
+                        totalItemPrice.addAndGet(GetitemPrice);
                         String itemPrice = NumberFormat.getNumberInstance(Locale.US).format(GetitemPrice);
                         equipmentPanels.add(new ItemPanel(itemComposition, kitType, itemImage, itemPrice), c);
                         c.gridy++;

--- a/src/main/java/equipmentinspector/EquipmentInspectorPlugin.java
+++ b/src/main/java/equipmentinspector/EquipmentInspectorPlugin.java
@@ -9,6 +9,7 @@ import net.runelite.api.Client;
 import net.runelite.api.ItemComposition;
 import net.runelite.api.Player;
 import net.runelite.api.kit.KitType;
+import net.runelite.client.game.ItemManager;
 import net.runelite.client.menus.MenuManager;
 import net.runelite.client.plugins.Plugin;
 import net.runelite.client.plugins.PluginDescriptor;
@@ -67,6 +68,9 @@ public class EquipmentInspectorPlugin extends Plugin
 
 	@Inject
 	private Notifier notifier;
+
+	@Inject
+	private ItemManager itemManager;
 
 	@Override
 	protected void startUp() throws Exception
@@ -132,6 +136,7 @@ public class EquipmentInspectorPlugin extends Plugin
 				{
 					Player p = targetPlayer.get();
 					Map<KitType, ItemComposition> playerEquipment = new HashMap<>();
+					Map<KitType, Integer> equipmentPrices = new HashMap<>();
 
 					for (KitType kitType : KitType.values())
 					{
@@ -140,9 +145,10 @@ public class EquipmentInspectorPlugin extends Plugin
 						{
 							ItemComposition itemComposition = client.getItemDefinition(itemId);
 							playerEquipment.put(kitType, itemComposition);
+							equipmentPrices.put(kitType, itemManager.getItemPrice(itemId));
 						}
 					}
-					equipmentInspectorPanel.update(playerEquipment, playerName);
+					equipmentInspectorPanel.update(playerEquipment, equipmentPrices, playerName);
 				}
 		}
 	}

